src/behavior.scm

```diff
@@ -203,6 +203,29 @@
 			(ListLink (Node "--- Glance at new person")))
 	))
 
+;; New recognized person sequence
+(DefineLink
+	(DefinedPredicate "Interacting Sequence for recognized person")
+	(SequentialAnd
+		;(True (DefinedSchema "glance at new person"))
+		(True (Put ; Setting interaction target for "look at person"
+			(DefinedPredicate "Set interaction target")
+			(RandomChoice (Put; FIXME Replace by multiple face tracking.
+				(DefinedSchemaNode "Get recognized face's face id")
+				(DefinedSchema "Get recognized faces")))))
+		(DefinedPredicate "look at person")
+		(True (Put
+			(DefinedPredicate "Greet recognized person")
```

Comment:
1. Recognized flag is set through https://github.com/opencog/ros-behavior-scripting/blob/master/face_track/face_track.py#L416
2. The rule added is the one that decides to say hello. It doesn't decide to not say hello, but since the the condition for the `(DefinedPredicate "Did someone arrive?")`  rule of the same demands is also satisfied, and the rules have the same weight, the new rule has 50% chance of be selected.
3. I don't get how this is a short circuit of the nlp pipeline. AFAIK we haven't experimented with sureal including non linguistic information. Even if we decide to use self-model information we have to deal with normalizing the representation from StateLinks to PLN-format as nlp-pipeline requires that.